src/components/table/EditView.tsx

```diff
@@ -37,10 +39,16 @@ const EditView: FC<EditViewProps> = ({ task, setIsEditing }) => {
 
   const onSubmit = (values: Inputs) => {
     const editedTask = { ...task, title: values.title, description: values.description }
-    if (values) {
-      editTask(editedTask);
-      navigate('/yourTasks')
-    }
+    setDialogConfiguration({
+      onSubmit: async () => {
+        editTask(editedTask)
+        navigate('/yourTasks')
```

Comment:
to dziala bez `.then`?

Reply:
działa :) sprawdzałem przed commitem i teraz bo aż zwątpiłem ale działa